lib/src/main/kotlin/at/bitfire/ical4android/AndroidEvent.kt

```diff
@@ -241,7 +244,7 @@ class AndroidEvent(
                 if (duration is Duration)
                     duration = Period.ofDays(duration.toDays().toInt())
                 tsEnd = (startDate + duration).toEpochDay() * TimeApiExtensions.MILLIS_PER_DAY
-                duration = null
+                //duration = null
```

Comment:
I don't think setting duration to null here does anything so I am not sure why it was there in the first place. Why comment it out instead of deleting it?

Reply:
As far as I remember, it was there because if `duration` _would_ be used later, we would have to set it to null here.

By commenting it out we don't forget that setting it to null might be something we want to do.

However this code will be refactored with the new mapper anyway.